### PR TITLE
Added basic PLOD visualization

### LINKED ISSUE #117: Presenting PLODs
Maybe this has been discussed in other issues but I couldn't find any ref. 
1. We need ideas on how to present PLODs. Whatever you suggest think intuitive, colorful, entertaining.
2. When presenting PLOD you should have some text (either on the page or in a pop hint) that explains that predictions are based on data from the books *_and not the show *_. then make a reference to the description of the relevant prediction method (will be supplied by the B groups).

### LINKED ISSUE #178: Character-Detail: PLOD Visualisation
_To visualize the % of death from model 1 (better model) we can use an image similar to http://www.clker.com/cliparts/U/H/t/l/l/W/rip-tombstone.svg, progress bar in red_

### PR DESCRIPTION
**Pull request by mammuth**
screen shot 2016-03-31 at 20 03 55

Closes #178 
Closes #117

### CODE REVIEW

**Comment by mammuth** [comment]
What do you think @jorjo @yasha?

**Comment by sacdallago** [comment]
I like the tomb at the end, extremely cool!
It actually got me to an idea... Can you put the progress bar in the tomb? Vertically in this case of course, kind of an info-graphic

**Comment by mammuth** [comment]
@sacdallago your suggestion reminds me of @goldbergtatyana ones in our meeting at monday. She suggested a visualization like linkedin if I remember correctly, with a tombstone that is filling itself red:
screen shot 2016-03-31 at 19 53 57
This indeed would look cool.

I guess making this looking good is probably much more effort, so I'd keep it like we have it for now. If we have time left we can still re-iterate the visualization. :relaxed:

**Comment by mammuth** [comment]
C'mon @yasha, the PR was just for catching feedback on the first draft, I was about to add more commits... 
If someone wants you to merge their stuff they will let you know. :wink:

Please stop always closing others PR's without communicating it.

**Comment by sacdallago** [comment]
Well.. @yashha is right... It's a pull _request_, not a pull _draft_ :wink: 
Don't open PR in the first place if they should not be reviewed... If you want to discuss items, rather put them on the relevant issue, like you did anyway :smile:

**Comment by mammuth** [comment]
I highly disagree! 
Pull request does not mean it needs to be merged immediately. 
If you think so, then Github could even just remove the whole "conversation" section?

I don't know how you guys use git methodology in your projects but I've never seen this kind of behavior, nor do I find it useful. 
(speaking of teams where there is no dedicated lead dev or project manager who is responsible for exactly this thing) 

Furthermore it's someone's pull request and this someone has a reason why it isn't merged yet. Maybe to speak about the changes with someone else in a public matter (oh wait, that's what the conversations are for, right).

In this specific case you see that in one of the issues Tatyana and me were speaking about the pages describing the algorithms and I told her I will add those pages and a markdown file for it. This is absolutely related to this PR. 

Next, when someone wants another one to merge their stuff - that's what the assigning functionality is for. There is no reason why anyone should merge PR's that they are not assigned to, nor are they related to the issue / implementation / whatever. 

To sum up my point of view on this:
- either the owner assignes another person to look into it / merge it
- he handles it himself
- there is a dedicated person for merging features 

Edit:
_Don't open PR in the first place if they should not be reviewed..._
Of course you're right on this. But review != someone who wasn't involved in the process around the feature, comes by and **merges** the PR without communicating at all :wink:

**Comment by kordianbruck** [comment]
@sacdallago sorry for mixing in here, but no you are wrong. In the github Workflow, a pull request is opened to let others know you are working on a couple issues on that specific Feature branch. Please refer to the github help: https://help.github.com/articles/using-pull-requests/

I often open a PR to get feedback for the proposed changes. Usually another employee reviews them and a third party merges the changes. 

My 2 cents.

Edit: also to see all the tests ran through it gives you a nicer GUI ;-)

**Comment by sacdallago** [comment]
Well, I see your points and I can't disagree, but as @mammuth mentioned it depends on habits. For me PRs are: Feature is done, I need comments on the code and on the feature, but if everything is alright I expect it to get merged; aka: I ignore it and move to something else. It is not just _my way of thinking_ but I like the _meaning_ of words. _Request_ is a request, not a draft. I'm not blaming you, don't get me wrong, it's just that you can't call something pull request and then promote discussions about future work. It should be only to revise the _current state_ and review the code.

If this was your intention @mammuth , then @yashha was too hasty merging. But on the other hand, if your code is fully functional and you are only talking about an _enhancement_, that is already another feature and PR (ideally also a different branch).

GitHub, in my opinion, is not managing the whole feature-model idea right anyway. I was working mainly with Atalassian and that's a much nicer suite. You would really just open PR for the sake of the PR to be reviewed and merged, or rejected and closed. Definitively not to have discussions (there were plenty other places and means to do that)... And you can assign >1 person to a PR or **issue**(what a name that is for tasks, then), which in that case are Tasks, User Stories, blablabla... makes so much more sense.
Also: direct branching from Tasks/Issues?! Com'on that's like 101 to me, I really don't get how GitHub can be so behind on these things.

That there is no dedicated lead dev or project manager.. I don't know how much that applies in open source communities! You don't get one PM or the lead dev, you just have previous contributors or other people that act as maintainers from time to time, and they definitively didn't code with you :) It's a joint effort. Sure, at the end of the day there is Linus Torvalds who throws shit on whoever implemented a feature 6mo earlier and the asshole that merged it into the kernel 5mo before, but [that's a different story](http://thenextweb.com/dd/2015/11/02/linux-creator-linus-torvalds-had-a-meltdown-over-a-pull-request-and-it-was-awesome/#gref).
And if it brings you any comfort: I am reading through most of your PRs anyway (code included) the way I expect PRs to be (as described above) and I have been commenting on stuff I didn't want to see in the code. I also reverted some merges I didn't like :smiling_imp: But if I see that you guys are fighting over who closes PRs I need to state what **I** expect PRs to be :laughing: So that @yashha doesn't get the blame if it is me who is setting the rules here :dancer: 

Yes, yes: I am the awful and [ignorant](https://www.youtube.com/watch?v=BHqgHFcmAOc) PM if you want, I can live with that :laughing: and you the poor developers that have to go against your well-taught practices :laughing:
I'm sorry it has to be like this, truly, but life is tough.

I am happy anyway that we all see each others points of view and I think this discussion benefited us not only with the _same old_ code-related matters, but a little beyond to philosophical somewhat _religious_ beliefs :)

**Comment by sacdallago** [comment]
Ps. @kordianbruck your link is supporting my thesis?!

> When you're ready to submit your pull request, click Create pull request.

and after that

> After reviewing the basic description, commits, and cumulative diff, the person tasked with applying the changes may have questions or comments. Perhaps the coding style doesn't match project guideline, or the change is missing unit tests, or maybe everything looks great and some props are in order. The discussion view encourages and captures this type of discussion.

Also the screenshot after this last quote is quite exactly what a PR is in my opinion: finished features. I think here we got in the subtle discussion about: is this code fully functional and if yes then the "next thing" to do is an enhancement, aka: new feature. Very subtle!

**Comment by kordianbruck** [comment]
@sacdallago well it can support either view as there is no dedicated person tasked with applying changes. But usually I would say it is the person opening the PR unless someone else is assigned :tada:
